sheets/api.py

```diff
@@ -24,7 +23,7 @@
 import pygsheets
 from googleapiclient.errors import HttpError
 
-from mitxpro.utils import now_in_utc, get_error_response_summary
+from mitxpro.utils import now_in_utc
```

Comment:
Weird that pylint didn't catch this; it should have flagged this as `unused-import`, right?  I'll create an issue for it.

Reply:
https://github.com/mitodl/mitxpro/pull/2055